backpack/custom_module/graph_utils.py

```diff
@@ -0,0 +1,201 @@
+"""Transformation tools to make graph BackPACK compatible."""
+from copy import deepcopy
+from typing import Set
+
+from torch.fx import Graph, GraphModule, Node, Tracer
+from torch.nn import Flatten, Module
+
+from backpack.custom_module.branching import ActiveIdentity, Branch, SumModule
+from backpack.custom_module.scale_module import ScaleModule
+from backpack.utils import TORCH_VERSION_AT_LEAST_1_9_0
+
+
+class BackPackTracer(Tracer):
+    """This custom tracer recognizes BackPACK custom modules."""
+
+    def is_leaf_module(self, m, module_qualified_name):  # noqa: D102
+        if isinstance(m, (ScaleModule, SumModule, Branch, ActiveIdentity)):
+            return True
+        else:
+            return super().is_leaf_module(m, module_qualified_name)
+
+
+def print_table(module: Module) -> None:
+    """Prints a table of the module.
+
+    Args:
+        module: module to analyze
+    """
+    graph_resnet18 = BackPackTracer().trace(module)
+    graph_resnet18.print_tabular()
```

Comment:
The name suggests there is a ResNet18 in here. Can you clarify this?

Reply:
It's an old name.
I renamed it from `graph_resnet18` to `graph`.